carl/modules/codeableconcept.py

```diff
@@ -29,8 +29,8 @@ def value_param(self):
         """
         if len(self.codes) != 1:
             raise ValueError(
-                f"Require single coding to include {self} in search for value_param ")
+                f"Require single coding to include {self} in search for value_param "
```

Comment:
Is this trailing space intentional?...
```suggestion
                f"Require single coding to include {self} in search for value_param"
```